### PR TITLE
Migration to package format 2

### PR DESCRIPTION
**Pull request by fmessmer**

### CODE REVIEW

File: cob_gazebo_worlds/package.xml [hunk 1]
```diff
@@ -13,23 +13,23 @@
...
-  <run_depend>tf</run_depend>
-  <run_depend>velocity_controllers</run_depend>
-  <run_depend>xacro</run_depend>
+  <exec_depend>controller_manager</exec_depend>
+  <exec_depend>gazebo_msgs</exec_depend>
+  <exec_depend>gazebo_ros_control</exec_depend>
+  <exec_depend>gazebo_ros</exec_depend>
+  <exec_depend>joint_state_controller</exec_depend>
+  <exec_depend>joint_state_publisher</exec_depend>
+  <exec_depend>kdl_parser</exec_depend>
```
Comment (NadiaHG) [inline cob_gazebo_worlds/package.xml]: Do we need kdl_parser?

**Comment by mgruhler** [comment]
:ok:

File: cob_gazebo_worlds/package.xml (hunk 1 above)
Comment (fmessmer) [inline cob_gazebo_worlds/package.xml]: removed

**Comment by NadiaHG** [comment]
LGTM

**Pull request merged**